deploy/auto-install.sh

```diff
@@ -55,7 +55,7 @@ error_msg_with_continue() {
 apt_dependenices_setup() {
 	start_step "Setting up dependencies..."
 	apt --yes install python3 python3-pip git python3-dev gawk libffi-dev libssl-dev gcc make &>>$LOG_FILE
-	pip3 install --upgrade pip &>>$LOG_FILE
+	#pip3 install --upgrade pip &>>$LOG_FILE
```

Comment:
Why this change?

Reply:
APT now manages the python environment not pip

Reply:
Let's just remove the line.
```suggestion
```